scripts/musicbrainz-initial-dump/importArtists.js

```diff
@@ -0,0 +1,77 @@
+//NEXT STEPS:
+//2. be sure duplicates are not saved
+//5. test in lambda
+//6. execute locally
+
```

Comment:
I'm not sure these files will continue to live in `scripts/musicbrainz-initial-dump`. There will eventually be several scripts that all run as AWS Lambda functions, so I have to figure out what the best folder structure is to pull that off.